code/core/src/core-server/utils/StoryIndexGenerator.ts

```diff
@@ -439,10 +439,11 @@ export class StoryIndexGenerator {
 
         const id = input.__id ?? toId(input.metaId ?? title, storyNameFromExport(input.exportName));
         const tags = combineTags(...projectTags, ...(input.tags ?? []));
+        const subtype = input.type !== 'story' ? 'story' : (input.subtype ?? 'story');
```

Comment:
Can you verify whether this only applies subtype story to story entries, and not stuff like docs entries? If true, LGTM!